.travis.yml

```diff
@@ -12,38 +12,34 @@ jobs:
       name: test-osx
       os: osx
       install:
+        - nvm install 8
+        - nvm use 8
+        - npm install -g yarn
         - rustup toolchain install nightly-2018-07-17
         - rustup component add rustfmt-preview --toolchain nightly-2018-07-17
       before_script:
         - cargo fetch --verbose
       script:
-        - cargo +nightly-2018-07-17 fmt -- --check && RUST_BACKTRACE=1 cargo test --verbose --all
+        - cargo +nightly-2018-07-17 fmt -- --check
```

Comment:
AFAIK, It uses `&&` since Travis runs the next script even the previous script failed.
Would you check it?

Reply:
https://github.com/sgkim126/travis-test/commit/4edea17a19845da5cecc451e28019d0c1f7227bc
https://travis-ci.org/sgkim126/travis-test/jobs/440060659 Yes it does.

Reply:
I think it should be in a one line, for fast termination. I'll fix this.

Reply:
I'll fix this using a flag env variable